### PR TITLE
Feature flu various robots

### PR DESCRIPTION
**Pull request by joachimhgg**
**Protos**

- [x] (749) robots/neuronics/ipr/protos/IprHd6ms90.proto
- [x] (750) robots/neuronics/ipr/protos/IprHd6m90.proto
- [x] (751) robots/neuronics/ipr/protos/IprHd6m180.proto
- [x] (752) robots/neuronics/ipr/protos/IprHd6ms180.proto

**Worlds**

- [x] (224) projects/robots/fp_robotics/p-rob3/worlds/p-rob3.wbt
- [x] (245) projects/robots/robotcub/icub/worlds/icub_stand.wbt
- [x] (002) scripts/icon_studio/worlds/icon_studio.wbt and objects.json
- [x] (268) projects/robots/micromagic/scout/worlds/scout.wbt
- [x]  (299) projects/robots/neuronics/ipr/worlds/ipr_factory.wbt
- [x] (300) projects/robots/neuronics/ipr/worlds/ipr_cube.wbt
- [x] (301) projects/robots/neuronics/ipr/worlds/ipr_collaboration.wbt
- [x] (302) projects/robots/neuronics/ipr/worlds/ipr_models.wbt

### CODE REVIEW

File: scripts/icon_studio/controllers/icon_creator/objects.json [hunk 1]
```diff
@@ -6,962 +6,1680 @@
...
+      1.0,
+      1.0
+    ],
+    "fields": "  rotation 0.0 0.0 1.0 0.3",
+    "nodeString": "  Cypress{ rotation 0.0 0.0 1.0 0.3 }"
   },
   "projects/objects/trees/protos/Forest": {
-    "background": [1.0, 1.0, 1.0],
-    "fields": "translation 0 0.1 0 rotation 0 1 0 0.3"
+    "background": [
```
Comment (ad-daniel) [inline scripts/icon_studio/controllers/icon_creator/objects.json:1422]: Forest has a bug, likely an issue with the PROTO itself.
a

File: scripts/icon_studio/controllers/icon_creator/objects.json [hunk 2]
```diff
@@ -6,962 +6,1680 @@
...
+    "nodeString": "  Robotino3SickMountingSupport{ rotation -0.8628592318041438 -0.35740509601564097 0.3574010960145665 1.7177653071795864 }"
   },
   "projects/robots/festo/robotino3/protos/Robotino3Webcam.proto": {
-    "fields": "translation 0 -0.208 0 rotation -0.18616 0.69475 0.69475 2.7735"
+    "fields": "  translation 0 0 -0.208 rotation -5.94733999978894e-06 5.966989999788244e-06 0.999999999964512 1.0471953071795863",
+    "nodeString": "  Robotino3Webcam{ translation 0 0 -0.208 rotation -5.94733999978894e-06 5.966989999788244e-06 0.999999999964512 1.0471953071795863 }"
   },
   "projects/robots/festo/robotino3/protos/Robotino3WebcamMountingSupport.proto": {
-    "fields": "translation 0 -0.211 0 rotation -0.28109 0.6786 0.6786 2.59356"
+    "fields": "  translation 0 0 -0.211 rotation 0.6785976608971787 -0.2810890310957679 -0.6785972083810249 -2.59356",
```
Comment (ad-daniel) [inline scripts/icon_studio/controllers/icon_creator/objects.json:1518]: this object should have a proper (monotone color) background like everything else

File: scripts/icon_studio/controllers/icon_creator/objects.json [hunk 3]
```diff
@@ -6,962 +6,1680 @@
...
+      1.0,
+      1.0
+    ],
     "colorThreshold": 0.001,
-    "nodeString": "Robot { translation 0 0.027 0 rotation -0.6087598420165832 1.0122819105744343e-6 -0.7933545580299642 3.1415961372248167 children [ Slot { endPoint TinkerbotsWheel { } } ] }"
+    "fields": "  rotation -0.13052795126003722 -0.9914446297883028 -1.947439272813855e-06 3.14159",
+    "nodeString": "  Robot { translation 0 0 0.027 rotation -0.13052795126003722 -0.9914446297883028 -1.947439272813855e-06 3.14159 children [ Slot { endPoint TinkerbotsWheel { } } ] }"
+  },
+  "projects/robots/pal_robotics/tiago++/protos/Tiago++.proto": {
+    "fields": "  rotation 0.11178796221525046 -0.1471636664273077 0.9827747996298206 1.6477654786391709",
```
Comment (ad-daniel) [inline scripts/icon_studio/controllers/icon_creator/objects.json:1654]: Tiago robot is backwards

File: scripts/icon_studio/controllers/icon_creator/objects.json [hunk 4]
```diff
@@ -6,962 +6,1680 @@
...
+    "nodeString": "  Tiago++{ rotation 0.11178796221525046 -0.1471636664273077 0.9827747996298206 1.6477654786391709 }"
   },
   "projects/robots/pal_robotics/tiago_base/protos/TiagoBase.proto": {
-    "fields": "rotation 1 0 0 -1.5708"
+    "fields": "  rotation -0.04339021564602396 -0.1791348245106523 0.9828672361177421 1.6039571495576197",
```
Comment (ad-daniel) [inline scripts/icon_studio/controllers/icon_creator/objects.json:1658]: Tiago base is half-clipped into the ground

File: scripts/icon_studio/controllers/icon_creator/objects.json [hunk 5]
```diff
@@ -6,962 +6,1680 @@
...
+    "nodeString": "  TurtleBot3Burger{ rotation -9.589789993248847e-09 -2.315170000239143e-08 -0.9999999999999997 0.7853953071795863 }"
   },
   "projects/vehicles/protos/mercedes_benz/MercedesBenzSprinter.proto": {
-    "fields": "rotation 0 1 0 2.618"
+    "fields": "  rotation 0.7745939356506115 0.4472169628474523 0.4472149628476184 1.8234853071795867",
+    "nodeString": "  MercedesBenzSprinter{ rotation 0.7745939356506115 0.4472169628474523 0.4472149628476184 1.8234853071795867 }"
   },
   "projects/vehicles/protos/mercedes_benz/MercedesBenzSprinterSimple.proto": {
-    "fields": "rotation 0 1 0 2.618"
+    "fields": "  rotation -0.7745939356506115 -0.4472169628474524 -0.4472149628476185 -1.8234853071795865",
```
Comment (ad-daniel) [inline scripts/icon_studio/controllers/icon_creator/objects.json:1682]: this one too

**Review by ad-daniel — CHANGES_REQUESTED** [review]
Mostly minor stuff, thank you

File: scripts/icon_studio/controllers/icon_creator/objects.json [hunk 6]
```diff
@@ -6,962 +6,1680 @@
...
+      1.0,
+      1.0,
+      1.0
+    ],
+    "fields": "  rotation 0.0 0.0 1.0 0.3",
+    "nodeString": "  Tree{ rotation 0.0 0.0 1.0 0.3 }"
   },
   "projects/robots/boston_dynamics/spot/protos/Spot.proto": {
-    "fields": "translation -0.573555 -1.028 -0.0370218 rotation 0 1 0 -1.96308 controller \"void\""
+    "fields": "  translation 0.0370218 0.573555 -1.028 rotation -1.3505393835251057e-16 1.2211986311439658e-16 -1.0 1.96308 controller \"void\"",
```
[the remark this answers is not in the document]
Comment (joachimhgg) [reply scripts/icon_studio/controllers/icon_creator/objects.json:1485]: It is not failing when testing with a small dataset. I put in in the end of the json file.

**Comment by joachimhgg** [comment]
You can test the fixes by adding this code in your `objects.json`: 
```
{
  "default": {
    "colorThreshold": 0.01,
    "background": [1.0, 1.0, 1.0],
    "fields": "rotation 0 1 0 0.3",
    "nodeString": ""
  },  
  "projects/robots/pal_robotics/tiago_base/protos/TiagoBase.proto": {
    "nodeString": "  TiagoBase{ rotation 0 0 -1 1.57 }"
  },
  "projects/vehicles/protos/mercedes_benz/MercedesBenzSprinter.proto": {
    "nodeString": "  MercedesBenzSprinter{ rotation 0 0 1 3.14159 }"
  },
  "projects/vehicles/protos/mercedes_benz/MercedesBenzSprinterSimple.proto": {
    "nodeString": "  MercedesBenzSprinterSimple{ rotation 0 0 1 3.14159 }"
  },
  "projects/robots/boston_dynamics/spot/protos/Spot.proto": {
    "fields": "  translation 1.028 -0.0370218 0.573555 rotation 0 0 -1 1.96308 controller \"void\"",
    "nodeString": "  Spot{ translation 1.028 -0.0370218 0.573555 rotation 0 0 -1 1.96308 controller \"void\" }"
  },
  "projects/objects/balls/protos/Ball": {
    "fields": "   translation 0 0 0.0325 rotation 0.0 0.0 1.0 0.3",
    "nodeString": "     Ball{ translation 0 0 0.0325 rotation 0.0 0.0 1.0 0.3 }"
  },
  "projects/objects/balls/protos/SoccerBall": {
    "fields": "   translation 0 0 0.113 rotation 0.0 0.0 1.0 0.3",
    "nodeString": "     SoccerBall{ translation 0 0 0.113 rotation 0.0 0.0 1.0 0.3 }"
  },
  "projects/objects/geometries/protos/TexturedParallelepiped": {
    "fields": "   rotation 0.0 0.364549 0.931184 -0.561795",
    "nodeString": "   Transform { translation 0 0 0.06 rotation 0.0 0.364549 0.931184 -0.561795  children [ Shape { appearance Appearance { material Material { } texture ImageTexture { url [ \"textures/tagged_wall.jpg\" ] } } geometry TexturedParallelepiped { } } ] }"
  },
  "projects/objects/road/protos/LaneSeparation": {
    "fields": "   translation 0 0 0.01 rotation 0.0 0.0 1.0 1.5708 length 19",
    "nodeString": "   LaneSeparation{ translation 0 0 0.01 rotation 0.0 0.0 1.0 1.5708 length 19 }"
  },
  "projects/objects/road/protos/RoadLine": {
    "fields": "  translation 0 0.5 0.03 rotation 0.0 0.0 1.0 1.57",
    "nodeString": "    Road { translation 0 0.5 0.03 rotation 0.0 0.0 1.0 1.57 width 16 numberOfLanes 4 wayPoints [ 0 0 0 -20.0 -10.0 0 -50.0 10.0 0 ] lines [ RoadLine { type \"dashed\" } RoadLine { type \"continuous\" } RoadLine { type \"double\" color 1 1 0 } ] splineSubdivision 6 } "
  },
  "projects/robots/festo/robotino3/protos/Robotino3WebcamMountingSupport.proto": {
    "fields": "   translation 0 0 0.211 rotation 0.6785976608971788 -0.28108903109576794 -0.678597208381025 -2.59356",
    "nodeString": "   Robotino3WebcamMountingSupport{ translation 0 0 0.211 rotation 0.6785976608971788 -0.28108903109576794 -0.678597208381025 -2.59356 }"
  },
  "projects/robots/kinematics/tinkerbots/protos/TinkerbotsPivot": {
    "background": [
      1.0,
      1.0,
      1.0
    ],
    "colorThreshold": 0.001,
    "fields": " rotation 0 0 -1 0.261803",
    "nodeString": " Robot { rotation 0 0 -1 0.261803 children [ Slot { endPoint TinkerbotsPivot { } } ] }"
  },
  "projects/robots/pal_robotics/tiago++/protos/Tiago++.proto": {
    "fields": "  rotation -0.230732 -0.109357 0.966852 -1.27752",
    "nodeString": "  Tiago++{ rotation -0.230732 -0.109357 0.966852 -1.27752 }"
  }
}
```

**Comment by ad-daniel** [comment]
Very good, only 2 details remaining:

- forest on icon studio still showing up wrong

Issue coming from the other (previous) PR:
- when I load `visual_tracking.wbt` I get these errors:
```
WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF SIDE_RIGHT_INDICATOR Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/indicator.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/indicator.jpg
WARNING: DEF YELLOW_CAR Transform > Transform > DEF FOGLIGHT Transform > DEF FRONT_FOGLIGHT Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/antifog.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/antifog.jpg
WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF COACHWORK Shape > DEF VEHICLE_APP PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/matte_car_paint/matte_car_paint_base_color.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/matte_car_paint/matte_car_paint_base_color.jpg
WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF COACHWORK Shape > DEF VEHICLE_APP PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/glossy_car_paint/glossy_car_paint_roughness.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/glossy_car_paint/glossy_car_paint_roughness.jpg
WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF COACHWORK Shape > DEF VEHICLE_APP PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/glossy_car_paint/glossy_car_paint_normal.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/glossy_car_paint/glossy_car_paint_normal.jpg
WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF FRONT_PLATE Transform > Shape > DEF PLATE_APPEARANCE PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/plate.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/plate.jpg
WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF REAR_LOGO Transform > Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/citroen/textures/citroen_logo_base_color.png: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/citroen/textures/citroen_logo_base_color.png
WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF REAR_LOGO Transform > Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/citroen/textures/citroen_logo_normal.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/citroen/textures/citroen_logo_normal.jpg
WARNING: DEF YELLOW_CAR Transform > DEF FRONT_RIGHT_WHEEL Solid > Slot > Slot > Solid > Transform > DEF TIRE Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_base_color.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_base_color.jpg
WARNING: DEF YELLOW_CAR Transform > DEF FRONT_RIGHT_WHEEL Solid > Slot > Slot > Solid > Transform > DEF TIRE Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_roughness.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_roughness.jpg
WARNING: DEF YELLOW_CAR Transform > DEF FRONT_RIGHT_WHEEL Solid > Slot > Slot > Solid > Transform > DEF TIRE Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_normal.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_normal.jpg
WARNING: DEF YELLOW_CAR Transform > DEF FRONT_RIGHT_WHEEL Solid > Slot > Slot > Solid > Transform > DEF TIRE Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_occlusion.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_occlusion.jpg
WARNING: DEF BUS Transform > Transform > Group > DEF COACHWORK Shape > DEF APPEARANCE PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/matte_car_paint/matte_car_paint_roughness.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/matte_car_paint/matte_car_paint_roughness.jpg
```
the urls should be something like: `webots:// ...`. They were probably changed manually to test something.

**Comment by joachimhgg** [comment]
> Very good, only 2 details remaining:
> 
>     * forest on icon studio still showing up wrong
> 
> 
> Issue coming from the other (previous) PR:
> 
>     * when I load `visual_tracking.wbt` I get these errors:
> 
> 
> ```
> WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF SIDE_RIGHT_INDICATOR Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/indicator.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/indicator.jpg
> WARNING: DEF YELLOW_CAR Transform > Transform > DEF FOGLIGHT Transform > DEF FRONT_FOGLIGHT Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/antifog.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/antifog.jpg
> WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF COACHWORK Shape > DEF VEHICLE_APP PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/matte_car_paint/matte_car_paint_base_color.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/matte_car_paint/matte_car_paint_base_color.jpg
> WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF COACHWORK Shape > DEF VEHICLE_APP PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/glossy_car_paint/glossy_car_paint_roughness.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/glossy_car_paint/glossy_car_paint_roughness.jpg
> WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF COACHWORK Shape > DEF VEHICLE_APP PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/glossy_car_paint/glossy_car_paint_normal.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/glossy_car_paint/glossy_car_paint_normal.jpg
> WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF FRONT_PLATE Transform > Shape > DEF PLATE_APPEARANCE PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/plate.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/textures/plate.jpg
> WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF REAR_LOGO Transform > Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/citroen/textures/citroen_logo_base_color.png: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/citroen/textures/citroen_logo_base_color.png
> WARNING: DEF YELLOW_CAR Transform > Transform > Group > DEF REAR_LOGO Transform > Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/citroen/textures/citroen_logo_normal.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/citroen/textures/citroen_logo_normal.jpg
> WARNING: DEF YELLOW_CAR Transform > DEF FRONT_RIGHT_WHEEL Solid > Slot > Slot > Solid > Transform > DEF TIRE Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_base_color.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_base_color.jpg
> WARNING: DEF YELLOW_CAR Transform > DEF FRONT_RIGHT_WHEEL Solid > Slot > Slot > Solid > Transform > DEF TIRE Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_roughness.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_roughness.jpg
> WARNING: DEF YELLOW_CAR Transform > DEF FRONT_RIGHT_WHEEL Solid > Slot > Slot > Solid > Transform > DEF TIRE Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_normal.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_normal.jpg
> WARNING: DEF YELLOW_CAR Transform > DEF FRONT_RIGHT_WHEEL Solid > Slot > Slot > Solid > Transform > DEF TIRE Shape > PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_occlusion.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/vehicles/protos/abstract/textures/tire_texture_occlusion.jpg
> WARNING: DEF BUS Transform > Transform > Group > DEF COACHWORK Shape > DEF APPEARANCE PBRAppearance > ImageTexture: Cannot download https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/matte_car_paint/matte_car_paint_roughness.jpg: Error opening https://raw.githubusercontent.com/cyberbotics/webots/feature-flu-sample-default-robots/projects/appearances/protos/textures/matte_car_paint/matte_car_paint_roughness.jpg
> ```
> 
> the urls should be something like: `webots:// ...`. They were probably changed manually to test something.

Forest.proto has been fixed in #3768. I changed the image links, It should be fixed now. thanks!

**Review by ad-daniel — APPROVED** [review]
Thank you!

**Pull request merged**